src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -285,11 +287,21 @@ public void close() {
     }
 
     public void basicPublish(String exchange, String routingKey, BasicProperties props, byte[] body) throws IOException {
-        ChannelHolder holder = getChannelFor(PinId.forRoutingKey(routingKey));
+        ChannelHolder holder = getChannelFor(PinId.forRoutingKey(exchange, routingKey));
         holder.withLock(channel -> channel.basicPublish(exchange, routingKey, props, body));
     }
 
-    public SubscriberMonitor basicConsume(String queue, ManualAckDeliveryCallback deliverCallback, CancelCallback cancelCallback) throws IOException {
+    public String queueDeclare() throws IOException {
```

Comment:
Rename to `declareQueue`?

Reply:
This format is chosen according method name of RabbitMQ channel